appveyor.yml

```diff
@@ -117,7 +117,6 @@ install:
 - if not "%WITHLANG%"=="" set SWIGWITHLANG==%WITHLANG%
 - if not "%WITHLANG%"=="" where %WITHLANG%
 - if "%SWIGLANG%"=="python" set PY3=3
-- if "%PY2%"=="2" set PY3=
```

Comment:
It seems silly to keep `PY3` which is now always set to `3` for Python.  It looks like the root of this is that `configure` takes `--with-python3` but the make rules are e.g. `check-python-version`.  The GHA config also has to deal with this.

Maybe `--with-python` should be an alias for `--with-python3` now.  This could reasonably be done as a follow-on clean-up though, as this change is already rather substantial.

Reply:
Nor is `configure.ac`.